mobile_verifier/migrations/22_coverage_objects.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE coverage_objects (
+    uuid UUID NOT NULL,
+    radio_type radio_type NOT NULL,
+    radio_key TEXT NOT NULL,
+    indoor BOOLEAN NOT NULL,
+    coverage_claim_time TIMESTAMPTZ NOT NULL,
+    trust_score INTEGER,
```

Comment:
isn't trust score a float? or is the intention to store it as a whole number multiple of the functional trust score?

Reply:
we receive it as an integer so I was just planning on storing it as one and convert to float when needed.

Reply:
:+1: